### DISCUSSION TITLE
Please make it possible to turn off TinyMCE for the admin

### DISCUSSION THREAD

**Discussion by buninsan** [mindstellar/shopclass#361] [General]
Please make it possible to turn off TinyMCE for the admin.
The alert-form-old.php functionality still doesn't work.

**Comment by navjottomer** [comment]
Okay, I'll add an option in the backend for TinyMCE, is there any specific location?
alert-form-old.php? I didn't understand.

**Reply by buninsan** [reply]
On this page:
/oc-admin/index.php?page=items&action=settings
Enable TinyMCE on frontend
Enable TinyMCE on backend

About alert-form-old.php
Search subscription not working
Registered and unregistered users cannot subscribe.

**Comment by buninsan** [comment]
Hey,
I have been moderating ads for a long time and noticed a bug in the script.
Confusion occurs due to scripts that reload custom fields, this is especially noticeable if you edit ads in several browser tabs.
The header information and custom fields in editable ads are arbitrarily changed.

**Comment by buninsan** [comment]
The new version of the script has a problem with custom fields
https://www.dropbox.com/s/b2y9qlxdjedi9ur/Video_2021-11-04_214317.wmv?dl=0
